### PR TITLE
chore(greenkeeper): upgrade to webpack 4.23.1

### LINKED ISSUE #655: An in-range update of webpack is breaking the build 🚨
## The devDependency [webpack](https://github.com/webpack/webpack) was updated from `4.22.0` to `4.23.0`.

🚨 [View failing branch](https://github.com/webpack/webpack-cli/compare/master...webpack:greenkeeper%2Fdefault%2Fwebpack-4.23.0).

This version is **covered** by your **current version range** and after updating it in your project **the build failed**.

webpack is a devDependency of this project. It **might not break your production code or affect downstream projects**, but probably breaks your build or test tools, which may **prevent deploying or publishing**.

<details>
<summary>Status Details</summary>

- ❌ **continuous-integration/travis-ci/push:** The Travis CI build could not complete due to an error ([Details](https://travis-ci.org/webpack/webpack-cli/builds/445789322?utm_source=github_status&utm_medium=notification)).
</details>

---

<details>
<summary>Release Notes for v4.23.0</summary>

<h1>Features</h1>
<ul>
<li>add <code>watchMode</code> flag to Compiler to be able to detect watch mode in plugins</li>
<li>Prefer chunk names of entrypoints when merging chunks</li>
<li>add <code>removedFiles</code> property to Compiler to detect removed files</li>
</ul>
<h1>Bugfixes</h1>
<ul>
<li>publish declarations to npm</li>
<li>upgrade <code>@webassemblyjs/*</code> for bugfix</li>
<li>fix crash when using a side-effect-free wasm module in production mode</li>
</ul>
<h1>Internal changes</h1>
<ul>
<li>test on node.js 12</li>
<li>fix memory leak in test suite</li>
</ul>
</details>

<details>
<summary>Commits</summary>
<p>The new version differs by 35 commits.</p>
<ul>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/a4feb6e15ff7ac19bab1a458de369a887dde44fe"><code>a4feb6e</code></a> <code>4.23.0</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/298fa4c9811f24bfc2d73315dc5fe7a6a609e130"><code>298fa4c</code></a> <code>Merge pull request #8266 from ljqx/wasm-finalize-exports-fix</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/035ceb6b1b9b14fe7c3c0da2738cd5d5e2ecf24b"><code>035ceb6</code></a> <code>force travis rerun</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/735b4aa56f712d36e62eec94731f3b0bd012be65"><code>735b4aa</code></a> <code>Merge pull request #8182 from xtuc/ci-node-nightly</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/6844c7526fe9431c1d59c855c9eb93787d41083e"><code>6844c75</code></a> <code>add test</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/82562497035541e9f13fe54f7707782602c5777b"><code>8256249</code></a> <code>longer timeouts for Compiler test</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/f7ff9139f0c815f007bd32bc53bad5a3f6e98424"><code>f7ff913</code></a> <code>use latest node 12 nightly</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/1c44f95b8c1851eb088a9b3e5944fee6e1f23ddd"><code>1c44f95</code></a> <code>fix bug where module is referenced which is not in the graph</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/98d077d0b63aae7854467bf77c727efff6d0ea22"><code>98d077d</code></a> <code>order values with different types correctly</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/f5e45bec2e27c64966ca8c2e5375cac9c2b21501"><code>f5e45be</code></a> <code>also export WebAssembly.Global when reexporting globals</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/b22f027f79d71fa700129bacaf075beac90c7f9a"><code>b22f027</code></a> <code>Update tests for new WebAssembly.Global API</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/edcadbb0a107cf73010395b3fdd8ca65e66a11ad"><code>edcadbb</code></a> <code>ci: add node 11</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/91755a2bdce523ac7bce05cfd5470b9d468c1926"><code>91755a2</code></a> <code>Merge pull request #8265 from webpack/ci/fix-memory-issue</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/523ddd7220b837af409fdbd4e0d5ffe10b81f863"><code>523ddd7</code></a> <code>avoid memory leak</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/48656d22b98a23237d314a6397c3d82d659a32fd"><code>48656d2</code></a> <code>[WasmFinalizeExportsPlugin] check if <code>ref</code> is null before using it</code></li>
</ul>
<p>There are 35 commits in total.</p>
<p>See the <a href="https://urls.greenkeeper.io/webpack/webpack/compare/894eca20bd678ca5473cfca28f332a7322c64b8e...a4feb6e15ff7ac19bab1a458de369a887dde44fe">full diff</a></p>
</details>

<details>
<summary>FAQ and help</summary>

There is a collection of [frequently asked questions](https://greenkeeper.io/faq.html). If those don’t help, you can always [ask the humans behind Greenkeeper](https://github.com/greenkeeperio/greenkeeper/issues/new).
</details>

---

Your [Greenkeeper](https://greenkeeper.io) Bot :palm_tree:

### PR DESCRIPTION
**Pull request by dhruvdutt**
**What kind of change does this PR introduce?**
Greenkeeper update

**Did you add tests for your changes?**
No

**If relevant, did you update the documentation?**
No

**Summary**
#655 Greenkeeper did not create a PR since the initial build failed and after updating to the newer version, it did not create a PR automatically, only updated the code on its branch.

**Does this PR introduce a breaking change?**
No

**Other information**
Closes #655

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
     "yargs": "^12.0.2"
   },
   "peerDependencies": {
-    "webpack": "^4.x.x"
+    "webpack": "^4.23.1"
```
Comment (dhruvdutt) [inline package.json]: I think we should revert this change.

**Comment by ev1stensberg** [comment]
Open a fre
